kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/CtlConf.scala

```diff
@@ -47,6 +47,22 @@ object CtlConf {
       .stringConf
       .createOptional
 
+  val CTL_REST_CLIENT_CONNECTION_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.ctl.rest.connection.timeout")
+      .doc("The timeout[ms] for establishing the connection with the kyuubi server.")
+      .version("1.6.0")
+      .timeConf
+      .checkValue(_ > 0, "must be positive number")
+      .createWithDefault(Duration.ofSeconds(30).toMillis)
+
+  val CTL_REST_CLIENT_SOCKET_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.ctl.rest.socket.timeout")
+      .doc("The timeout[ms] for waiting for data packets after connection is established.")
+      .version("1.6.0")
+      .timeConf
+      .checkValue(_ > 0, "must be positive number")
+      .createWithDefault(Duration.ofSeconds(30).toMillis)
+
   val CTL_REST_CLIENT_REQUEST_MAX_ATTEMPTS =
```

Comment:
could you also help add `ConfigEntry` for CTL_REST_CLIENT_REQUEST_MAX_ATTEMPTS, CTL_REST_CLIENT_REQUEST_ATTEMPT_WAIT and CTL_BATCH_LOG_QUERY_INTERVAL?

thanks